packages/discovery-react-components/src/components/SearchFacets/SearchFacets.tsx

```diff
@@ -49,6 +49,10 @@ interface SearchFacetsProps {
    * Number of facet terms to show when list is collapsed
    */
   collapsedFacetsCount?: number;
+  /**
+   * Exposed onChange function for external use
```

Comment:
yes this is good - i would slightly modify the javadoc here to indicate which onChange event this corresponds to. something like

> custom handler invoked when any <input> element changes in this component